internal/lookout/ui/src/containers/JobsContainer.tsx

```diff
@@ -109,6 +109,16 @@ class JobsContainer extends React.Component<JobsContainerProps, JobsContainerSta
           defaultFilter: [],
           width: 0.5,
         },
+        {
+          id: "jobStateDuration",
+          name: "Time in State",
+          accessor: "jobStateDuration",
+          urlParamKey: "job_state_duration",
+          isDisabled: false,
+          filter: [],
+          defaultFilter: [],
+          width: 0.7,
```

Comment:
@carlocamurri told me that all these width values need to add up to 1.

Reply:
I'll adjust

Reply:
I was wrong.  JobSet colums add to 1.  Looks like Jobs don't.